addTourpackageProcess.php

```diff
@@ -0,0 +1,167 @@
+<?php
+require 'connection.php';
+
+$jsonText = $_POST["tourPackageData"];
```

Comment:
You must validate this `tourPackagedata` parameter before assigning to a variable. It has some possibility that the parameter does not set to associated array or it can be the empty parameter.

Reply:
OK